test/utils.py

```diff
@@ -934,3 +944,34 @@ def run_with_config(self, config):
             with self.ini_path.open("w") as f:
                 f.write(config_old)
             self.admin("RELOAD")
+
+
+class OpenLDAP:
+    def __init__(self, config_dir):
+        self.port_lock = PortLock()
+        self.config_dir = config_dir
+        self.slapd_pid_file = self.config_dir / "ldap" / "slapd.pid"
+
+    def startup(self):
+        print(
```

Comment:
What is the purpose of this print here?

Reply:
This is for debug purpose, because for freebsd, it always error out cannot find the "start_openldap_server.sh" file I added. I don't know the reason. I forgot to remove it, so I will delete the print later.